app/konnect/runtime-manager/gateway-config.md

```diff
@@ -92,16 +92,13 @@ for all configuration options.
 
 The **Certificates** configuration page lists public certificates
 that enable encrypted requests and peer certification validation.
-
-This configuration page _does not_ manage runtime instance certificates. If you
-need to update a runtime instance certificate, see
-[Renew Certificates](/konnect/runtime-manager/runtime-instances/renew-certificates).
-
 A certificate object represents a public certificate and can be paired with a
 corresponding private key. Certificates handle SSL/TLS termination for encrypted
 requests, and can be used as a trusted CA store when validating the peer
 certificate of a client or service.
 
+You can manage runtime instance certificates from this page, including the creation, renewal, and removal of certificates, as well as uploading certificates for up to  16 certificates per runtime group. 
```

Comment:
Since there are two certificates tabs inside a Runtime Group, can we explicitly call out that data plane certificates can be found by navigating through the runtime group actions button in the UI?